xdsl/ir.py

```diff
@@ -1131,6 +1142,7 @@ def from_operation_list(ops: list[Operation]) -> Region:
         region.add_block(block)
         return region
```

Comment:
Do you have an idea how we could get rid of this as well? I think it might be fine as is, considering that this is a special case of a constructor essentially. What d you think?

Reply:
Yeah I'll deprecate this in a future PR. I don't like the skipping of levels we do sometimes, I feel that with APIs that are concise and reflect the semantics well we won't need quite as many helpers.

Reply:
And by future PR I mean when both Block and Region inits are in